content/open-source/sbom/getting-started-openvex-vexctl.md

```diff
@@ -15,7 +15,9 @@ toc: true
 terminalImage: academy/vexctl:latest
 ---
 
-The `vexctl` CLI is a tool to make VEX work. That is, you can use it to create, apply, and attest VEX (Vulnerability Exploitability eXchange) data. Its purpose is to help with the creation and management of VEX documents that allow "turning off" security scanner alerts of vulnerabilities known not to affect a product. Using VEX, software authors can communicate to their users that an otherwise vulnerable component has no security implications for their product.
+The `vexctl` CLI is a tool to make VEX work. That is, you can use it to create, apply, and attest VEX (Vulnerability Exploitability eXchange) data.
+
+The `vexctl` tool was built to help with the creation and management of VEX documents, communicate transparently to users as time progresses, and enabling the "turning off" of security scanner alerts of vulnerabilities known not to affect a given product. Using VEX, software authors can communicate to their users that an otherwise vulnerable component has no security implications for their product.
 
 This tutorial will walk you through some common commands in `vexctl`.
```

Comment:
There is a missing "you" or extraneous "run" in line 26/28 that starts "Using go..." 

For some reason, Github won't let me comment on this particular line, so I'm making a note of it here.